sdk/core/Azure.Core.Extensions/src/Azure.Core.Extensions.csproj

```diff
@@ -29,6 +29,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\identity\Azure.Identity\src\Azure.Identity.csproj" />
```

Comment:
Should this use a package reference rather than a project reference?

Reply:
I'll keep it a package reference until preview2 so we can iterate together.